app/src/main/java/org/mozilla/fenix/utils/Settings.kt

```diff
@@ -178,6 +184,17 @@ class Settings(private val appContext: Context) : PreferencesHolder {
         false
     )
 
+    /**
+     * Returns the current selected [Wallpaper].
+     */
+    fun getWallpaper(): Wallpaper {
+        return if (currentWallpaper.isEmpty()) {
+            Wallpaper.NONE
+        } else {
+            Wallpaper.valueOf(currentWallpaper)
+        }
+    }
```

Comment:
I agree with GL's offline comment that this might be better placed within the `Wallpaper` class itself, since there's already a property exposing the `currentWallpaper`.

Reply:
Just reposting my offline comment:

I am thinking what if `Wallpaper` was more of a manager in terms of managing the available wallpaper drawables, knowing what is the current wallpaper, next wallpaper to display, instead of the current enum class that it is.

Reply:
It probably means we might want `Wallpaper` to be part of `Components`, but can receive `Settings` as a parameter.